community-members.md

```diff
@@ -141,6 +141,14 @@ Maintainers:
 Maintainers:
 - [Francis Bogsanyi](https://github.com/fbogsany), Shopify
 
+## Proto
```

Comment:
I'd suggest we keep the same group for proto as we do for specs. 

@bogdandrutu would you agree?